pkg/spanconfig/spanconfigtestutils/utils.go

```diff
@@ -236,6 +240,19 @@ func PrintSpanConfigDiffedAgainstDefaults(conf roachpb.SpanConfig) string {
 	if !reflect.DeepEqual(conf.LeasePreferences, defaultConf.LeasePreferences) {
 		diffs = append(diffs, fmt.Sprintf("lease_preferences=%v", conf.VoterConstraints))
 	}
+	if !reflect.DeepEqual(conf.ProtectedTimestamps, defaultConf.ProtectedTimestamps) {
+		sort.Slice(conf.ProtectedTimestamps, func(i, j int) bool {
+			return conf.ProtectedTimestamps[i].Less(conf.ProtectedTimestamps[j])
+		})
+		var s string
```

Comment:
Instead of string concatenation in this form, we could collect an array of strings and use something like: `fmt.Sprint("pts=[%s]", strings.Join(timestamps, " ")`